<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="m_statusLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Status: Not running.</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_serialPortLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Serial port:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_waitResponseSpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_waitRequestLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Wait request, msec:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_requestLineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&quot;Hello, I'm Slave.&quot;</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_trafficLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>No traffic.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_runButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>30</y>
     <width>93</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_serialPortComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="m_responseLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Response:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
